superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -188,98 +191,105 @@ function ListViewCard({
   const Link = url && linkComponent ? linkComponent : AnchorLink;
   const theme = useTheme();
   return (
-    <StyledCard
-      data-test="styled-card"
-      cover={
-        cover || (
-          <Cover>
-            <Link to={url!}>
-              <div className="gradient-container">
-                <ImageLoader
-                  src={imgURL || ''}
-                  fallback={imgFallbackURL || ''}
-                  isLoading={loading}
-                  position={imgPosition}
-                />
-              </div>
-            </Link>
-            <CoverFooter className="cover-footer">
-              {!loading && coverLeft && (
-                <CoverFooterLeft>{coverLeft}</CoverFooterLeft>
-              )}
-              {!loading && coverRight && (
-                <CoverFooterRight>{coverRight}</CoverFooterRight>
-              )}
-            </CoverFooter>
-          </Cover>
-        )
-      }
-    >
-      {loading && (
-        <AntdCard.Meta
-          title={
-            <>
-              <TitleContainer>
-                <Skeleton.Input
-                  active
-                  size="small"
-                  css={{
-                    width: Math.trunc(theme.gridUnit * 62.5),
-                  }}
-                />
-                <div className="card-actions">
-                  <Skeleton.Button active shape="circle" />{' '}
-                  <Skeleton.Button
+    // <AntdThemeProvider theme={Object.assign(AtndTheme, {
+    //   Card:
+    //   colorBgContainerbackground: theme.grayscale.light4
+    // }}>
+    <ConfigProvider theme={listViewCardTheme}>
```

Comment:
This will always override the Card theme though. If we wanted a dark theme in the future, this would always be forced to `supersetTheme.colors.grayscale.light5`. So I guess we go back to considering the theme in Superset to be the controller, so that `supersetTheme.colors.grayscale.light5` will need to be changed to something that works on a dark theme, which begs the question "Will this work everywhere?". Hardly so, as there is no specification as of where light5 should be used in the design system I think.

Reply:
However, this is still better than custom styles as we will likely be targeting class names which should be more sensible to changes than theme configs.

Reply:
Agreed... using this config provider here is specifically to append/override styles for _that instance_ of the Card component. Other Cards higher up in the React/DOM tree will be left alone.

I agree that `light4` is going to be problematic, as sometimes that would actually be a dark color needed. Part of the theming SIP is to add "surface" colors (if I'm remembering the name right) and foreground/text colors, which would be more context-based and less shade-based. We'll have to replace these in a million places in the codebase, but doing it on AntD tokens will be a whole lot easier than doing it in CSS style overrides.